isis/src/galileo/apps/gllnims2isis/main.cpp

```diff
@@ -146,6 +146,7 @@ void importQubs(QString coreParamName, QString suffixParamName) {
   }
 
   // Create holder for original label
+  // TODO are we using origLabel anywhere?  It's not being written
   OriginalLabel origLabel(*pdsLabel);
```

Comment:
It looks like this can just be removed, it never gets passed to anything or used anywhere